frontend/src/employee-frontend/components/Header.tsx

```diff
@@ -207,23 +207,21 @@ const Header = React.memo(function Header({ location }: RouteComponentProps) {
                 </NavbarLink>
               </RequireRole>
 
-              {featureFlags.messaging &&
-                (featureFlags.experimental?.mobileDailyNotes ||
-                  hasPilotAccess) && (
-                  <RequireRole oneOf={['UNIT_SUPERVISOR', 'STAFF']}>
-                    <NavbarLink
-                      onClick={() => setPopupVisible(false)}
-                      className="navbar-item is-tab"
-                      to="/messages"
-                      data-qa="messages-nav"
-                    >
-                      {i18n.header.messages}
-                      {unreadCount > 0 && (
-                        <UnreadCount>{unreadCount}</UnreadCount>
-                      )}
-                    </NavbarLink>
-                  </RequireRole>
-                )}
+              {accounts.isSuccess && accounts.value.length > 0 && (
```

Comment:
I could be wrong, but doesn't every employee and group have an account?

Reply:
Yeah need to check with @mapehe, I'm not sure how this should work. The account list excludes group accounts outside pilot units, but if every emploee has a personal account then this won't work.